spec/formatting.md

```diff
@@ -140,6 +140,7 @@ from the surrounding text.
 To allow for _function handlers_ to ensure that certain _option_ values are set by _literals_,
 the _resolved value_ of each _option_ value MUST include information about 
 whether the _option_ value is a _literal_ or a _variable_.
+Note that this information is irrelevant for a _resolved value_ that is not used as the value of an _option_.
```

Comment:
I would do this differently?

```suggestion
For each _option_ value, the _resolved value_ MUST indicate if the value
was directly set with a _literal_, as opposed to being resolved from a _variable_.
This is to allow _functions handlers_ to require specific _options_ to be set using _literals_.

> For example, the _default functions_ `:number` and `:integer` require that the _option_
> `select` be set with a _literal_ (`plural`, `ordinal`, or `exact`). 
> Allowing a _variable_ in this _option_ would produce a _message_ that 
> is difficult to translate because the set of _keys_ is tied to the _selector_ chosen.

> [!NOTE]
> Such information is irrelevant for _resolved values_ not used as the value of an _option_.
```

Reply:
> is difficult to translate because the set of _keys_ is tied to the _selector_ chosen.
=>
is impossible to translate because the set of _keys_ is tied to the _selector_ chosen.

Reply:
@aphillips Accepted suggestion.